src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function convertToNumber(salary) {
+  const cleanedString = salary.replace(/[$,]/g, '');
+  const number = Number(cleanedString);
+
+  return number;
+}
+
+const ul = document.querySelector('ul');
+const liArr = [...ul.querySelectorAll('li')];
+
+function sortList(propName) {
+  liArr.sort((a, b) => {
+    const salaryB = convertToNumber(b.dataset[propName]);
+    const salaryA = convertToNumber(a.dataset[propName]);
+
+    return salaryB - salaryA;
+  });
+
+  ul.innerHTML = '';
+  liArr.forEach(li => ul.append(li));
+}
+
+sortList('salary');
```

Comment:
this function call should be moved below and place right before `getEmployees();` call, to line 39